tox.ini

```diff
@@ -85,31 +93,49 @@ deps =
     readme_renderer[md]
 commands = {posargs}
 
+[testenv:fmt]
+deps =
+    black
+    isort
+commands =
+    isort afkak
+    black afkak
+
+
 [flake8]
```

Comment:
:ghost: [Ruff](https://beta.ruff.rs/docs/) ruff ruff :ghost:

Reply:
🤯 will set this up in the next PR!! I saw we have pylint sitting there but it's not enabled? It should probably be a combination of pylint and Ruf.. since pylint can now catch things like https://pylint.readthedocs.io/en/latest/user_guide/messages/warning/logging-fstring-interpolation.html

Quick scan of the Ruff docs I don't see any support for this specifically (currently)

Reply:
I saw that when i was reviewing the PRs I was looking at it to see if i should pull it into my current work as well. I just did an initial setup of Ruff in #132